### PR TITLE
Adds ChromeOS, Linux GS pages

### LINKED ISSUE #10186: Create the Android page for macOS, Windows, Linux, ChromeOS

### LINKED ISSUE #10188: Create the Web page for macOS, Windows, Linux, ChromeOS
After the initial choice to start with Flutter, each OS+target page should offer an option to add web as a build target. The link will appear on these pages in the **Manage your Flutter SDK** section.

This will require 8 target pages that reuse the web setup includes from the web starting pages.

* Android on Linux
* Android on macOS
* Android on Windows
* Android on ChromeOS
* iOS on macOS
* Linux Desktop
* macOS Desktop
* Windows Desktop

### LINKED ISSUE #10191: Create the Linux page for web, Linux

### LINKED ISSUE #10255: Revise Linux & ChromeOS getting started pages to be consistent with other install pages
### Page URL

https://docs.flutter.dev/get-started/install/linux

### Page source

_No response_

### Describe the problem

The macOS and Windows getting started pages have been revised. The Linux and ChromeOS pages should provide the same experience.

### Expected fix

- [ ] Create two new root pages:
-- https://docs.flutter.dev/get-started/install/linux
-- https://docs.flutter.dev/get-started/install/chromeos
- [ ] Create branch pages for desktop, android, web.
- [ ] Link existing next step pages to these revised pages.

### Additional context

Relates to https://github.com/flutter/website/issues/10181

_No response_

### I would like to fix this problem.

- [X] I will try and fix this problem on docs.flutter.dev.

### PR DESCRIPTION
**Pull request by atsansone**
Adds ChromeOS, Linux GS pages plus all 2nd level add-ons to getting started.

Fixes #10255
Fixes #10186
Fixes #10188
Fixes #10191

Reviewed and approved by @anderdobo in #10302 

----

## New pages
These are mostly reusing includes or include templates from the other Getting Started pages. The main new pages are under:

- src/_includes/docs/install/reqs/linux/base.md
- src/get-started/install/chromeos/
- src/get-started/install/linux/
- src/platform-integration/android/install-android/
- src/platform-integration/linux/install-linux/
- src/platform-integration/web/install-web/

Changed to fix auto-population of current build artifacts in terminal code blocks.
- src/assets/js/archive.js

Added tabs
- src/assets/js/main.js

Refactored other framework pages to clean up includes.

----

## New page links
It will help to see these staged because of the heavy reuse and Liquid logic.

### ChromeOS install

- [Install on ChromeOS landing page](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/get-started/install/chromeos/)
- [Install Flutter with Android tools on ChromeOS](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/get-started/install/chromeos/android/)
- [Install Flutter with web tools on ChromeOS](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/get-started/install/chromeos/web/)

### Linux Install

- [Install on Linux landing page](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/get-started/install/linux/)
- [Install Flutter for Linux Desktop on Linux](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/get-started/install/linux/desktop)
- [Install Flutter with Android tools on on Linux](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/get-started/install/linux/android)
- [Install Flutter with web tools on Linux](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/get-started/install/linux/web)

### Add Linux desktop to Linux Android or Web install

- [Add Linux Desktop tools landing page](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/linux/install-linux/)
- [Add Linux Desktop tools to Flutter & Android tools on Linux](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/linux/install-linux/install-linux-from-android)
- [Add Linux Desktop tools to Flutter & Web tools on Linux](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/linux/install-linux/install-linux-from-web)

### Add Android to Linux Desktop install

- [Add Android tools to Flutter & Linux Desktop tools on Linux](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/android/install-android/install-android-from-linux)
- [Add Android tools to Flutter & Web tools on Linux](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/android/install-android/install-android-from-web-on-linux)

### Add Android to ChromeOS Web install

- [Add Android tools to Flutter on ChromeOS](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/android/install-android/install-android-from-web-on-chromeos)

### Add Web to Linux Desktop install

- [Add web tools to Flutter & Linux Desktop tools on ChromeOS](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/web/install-web/install-web-from-linux-desktop)
- [Add web tools to Flutter & Android tools on ChromeOS](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/web/install-web/install-web-from-android-on-linux)

### Add Web to ChromeOS Android install

- [Add web tools to ChromeOS](https://flutter-docs-prod--pr10316-fix-10255-ayluprrv.web.app/platform-integration/web/install-web/install-web-from-android-on-chromeos)

### CODE REVIEW

File: src/_includes/docs/install/accordions/install-chrome-from-cli.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+<details markdown="1">
```
Comment (sfshaza2) [inline src/_includes/docs/install/accordions/install-chrome-from-cli.md:2]: What's an "accordian"? Why is this directory called that?

File: src/_includes/docs/install/compiler/android.md [hunk 2]
```diff
@@ -21,8 +21,23 @@
...
+To create Android apps with Flutter, verify that the following Android
+components have been installed.
+
+* **Android SDK Platform, API {{ site.appnow.android_sdk }}**
+* **Android SDK Command-line Tools**
+* **Android SDK Build-Tools**
+* **Android SDK Platform-Tools**
+* **Android Emulator**
+
+If you haven't or don't know, continue with the following procedure.
```
Comment (sfshaza2) [inline src/_includes/docs/install/compiler/android.md:33]:
```suggestion
If you haven't installed these, or you don't know, continue with the following procedure.
```

File: src/_includes/docs/install/reqs/linux/unset-path.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{% assign terminal=include.terminal %}
+
+### Remove Flutter from your Windows Path variable
+{:.no_toc}
+
+To remove Flutter commands from {{terminal}},
+remove Flutter to the `PATH` environment variable.
```
Comment (sfshaza2) [inline src/_includes/docs/install/reqs/linux/unset-path.md:7]:
```suggestion
remove Flutter from the `PATH` environment variable.
```

File: src/platform-integration/linux/install-linux/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: Add Linux devtools for Flutter
+description: Configure your system to develop Flutter for Linux.
+short-title: Add Linux to possible Flutter SDK targets
```
Comment (sfshaza2) [inline src/platform-integration/linux/install-linux/index.md:4]: That's not much shorter! Still long for a tab.

File: src/platform-integration/web/install-web/install-web-from-android-on-chromeos.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: Add Web devtools to Flutter from Android on ChromeOS start
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-android-on-chromeos.md:2]:
```suggestion
title: Add Chrome DevTools to Flutter from Android on ChromeOS start
```

File: src/platform-integration/web/install-web/install-web-from-android-on-chromeos.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+description: Configure your system to develop Flutter web apps on ChromeOS.
+short-title: When you started with Android on ChromeOS
+---
+
+To add Web as a Flutter app target for ChromeOS with Android,
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-android-on-chromeos.md:7]: https://developers.google.com/style/word-list/#web
Also, it's DevTools, not devtools.

```suggestion
To add web as a Flutter app target for ChromeOS with Android,
```

File: src/platform-integration/web/install-web/install-web-from-android-on-linux.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+title: Add Web devtools to Flutter from Android on Linux start
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-android-on-linux.md:2]:
```suggestion
title: Add web DevTools to Flutter from Android on Linux start
```

File: src/platform-integration/web/install-web/install-web-from-android-on-linux.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+description: Configure your system to develop Flutter web apps on Linux.
+short-title: When you started with Android on Linux
+---
+
+To add Web as a Flutter app target for Linux with Android,
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-android-on-linux.md:7]:
```suggestion
To add web as a Flutter app target for Linux with Android,
```

File: src/platform-integration/web/install-web/install-web-from-linux-desktop.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+---
+title: Add Web devtools to Flutter from Linux start
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-linux-desktop.md:2]:
```suggestion
title: Add web DevTools to Flutter from Linux start
```

File: src/platform-integration/web/install-web/install-web-from-linux-desktop.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+description: Configure your system to develop Flutter web apps on Linux.
+short-title: When you started with Linux desktop
+---
+
+To add Web as a Flutter app target for Linux desktop, follow this procedure.
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-linux-desktop.md:7]:
```suggestion
To add web as a Flutter app target for Linux desktop, follow this procedure.
```

**Review by sfshaza2 — APPROVED** [review]
Other than some mostly minor comments, lgtm.

Some global changes:
- Check the wordlist. It's "web" not "Web".
- Also, it's "DevTools" not "devtools".

Also, some of your "short titles" are incredibly long. Can you shorten them further? Imagine a user having several open tabs in a page. Many of those would be very cropped.

Otherwise, lgtm, but please do fix "web" and "DevTools". And it's "Chrome DevTools or Flutter/Dart DevTools, not "web DevTools".

File: src/_includes/docs/install/accordions/install-chrome-from-cli.md (hunk 1 above)
Comment (atsansone) [reply src/_includes/docs/install/accordions/install-chrome-from-cli.md:2]: This is something that is hidden under

<details>
<summary>an accordion</summary>
Like this
</details>

File: src/platform-integration/linux/install-linux/index.md (hunk 4 above)
Comment (atsansone) [reply src/platform-integration/linux/install-linux/index.md:4]: True. I'll take a look at these. I wasn't happy with them either.

**Review by atsansone — COMMENTED** [review]
@Sfshaza2 I'm going to work on the titles before a merge.

File: src/_includes/docs/install/admonitions/install-in-order.md [hunk 11]
```diff
@@ -0,0 +1,3 @@
+{{site.alert.important}}
+  Perform this guide in sequence. Skipping steps can cause errors.
+{{site.alert.end}}
```
Comment (domesticmouse) [inline src/_includes/docs/install/admonitions/install-in-order.md:3]: Add NL at EOF

File: src/_includes/docs/install/compiler/xcode.md [hunk 12]
```diff
@@ -82,9 +82,9 @@ With Xcode, you can run Flutter apps on an iOS device or on the simulator.
 
 {% endif %}
 
-{% if time=="first" %}
+{% if attempt=="first" %}
 
-### Install CocoaPods
+### Install CocoaPods (Optional)
```
Comment (domesticmouse) [inline src/_includes/docs/install/compiler/xcode.md:87]: Optional is misleading. It's required if you use any package with native deps on iOS or macOS. I'd argue it's required for non trivial development on iOS

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**